devtools-ci-index.yaml

```diff
@@ -3479,4 +3479,17 @@
             ci_cmd: '/bin/bash cico_run_tests.sh'
             timeout: '20m'
             image_name: 'fabric8-analytics/f8a-hpf-insights'
-
+        - '{ci_project}-{git_repo}-fabric8-analytics-pylint':
+            git_repo: f8a-hpf-insights
+        - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
+            git_repo: f8a-hpf-insights
+        - '{ci_project}-{git_repo}-f8a-build-master':
+            git_organization: fabric8-analytics
+            git_repo: f8a-hpf-insights
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-analytics
+            deployment_units: 'f8a-hpf-insights'
```

Comment:
Deployment unit is `f8a-maven-hpf-insghts`: https://github.com/openshiftio/saas-analytics/pull/387/files#diff-977287510509ad69ac27ea400dda750bR5

Reply:
@msrb I want it to be `f8a-hpf-insights-maven` like we have for Kronos, ecosystem at the end. So I need to change the deployment_units to `f8a-hpf-insights-maven` in both?

Reply:
@msrb, the `deployement_units` will have` <ecosystem>`, what about ` deployment_configs`, it will only be repo name right?